crypto/src/test/java/org/springframework/security/crypto/bcrypt/BCryptTests.java

```diff
@@ -14,18 +14,33 @@
 
 package org.springframework.security.crypto.bcrypt;
 
-import org.junit.Test;
```

Comment:
Please do not change the import ordering unless it's necessary. 
It can make the revision history unclear as to when an import was added.